ledger/accountdb.go

```diff
@@ -275,7 +297,7 @@ func accountsDbInit(r db.Queryable, w db.Queryable) (*accountsDbQueries, error)
 	var err error
 	qs := &accountsDbQueries{}
 
-	qs.listAssetsStmt, err = r.Prepare("SELECT asset, creator FROM assetcreators WHERE asset <= ? ORDER BY asset desc LIMIT ?")
+	qs.listCreatablesStmt, err = r.Prepare("SELECT asset, creator, ctype FROM assetcreators WHERE asset <= ? AND ctype = ? ORDER BY asset desc LIMIT ?")
```

Comment:
Nit: you can probably drop `ctype` from the returned columns (e.g., `SELECT asset, creator FROM assetcreators ...`)